packages/account/src/Sections/Security/LoginHistory/__test__/login-history.spec.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { useLoginHistory, APIProvider } from '@deriv/api';
+import { isMobile } from '@deriv/shared';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import LoginHistory from '../login-history';
+
+jest.mock('@deriv/components', () => {
+    const original_module = jest.requireActual('@deriv/components');
+
+    return {
+        ...original_module,
+        Loading: jest.fn(() => 'mockedLoading'),
+    };
+});
+
+jest.mock('@deriv/api', () => ({
+    ...jest.requireActual('@deriv/api'),
+    useLoginHistory: jest.fn(),
+}));
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(() => false),
+}));
+
+const mockUseLoginHistory = useLoginHistory as jest.MockedFunction<typeof useLoginHistory>;
+
+describe('<LoginHistory />', () => {
+    let mock_store = mockStore({});
+    let response = {
+        data: {
+            formatted_data: [
+                {
+                    date: '',
+                    browser: '',
+                    action: '',
+                    status: '',
+                    ip: '',
+                    id: 0,
+                },
+            ],
+        },
+        isLoading: false,
+        isError: false,
+        error: '',
+    };
```

Comment:
```suggestion
    let response;
```

Reply:
I think we don’t need to crate an object, as you do it in beforeEach.
how do you think?
can we add types here?